cloudferry_devlab/cloudferry_devlab/generate_load.py

```diff
@@ -287,6 +299,65 @@ def update_filtering_file(self):
         self.switch_user(user=self.username, password=self.password,
                          tenant=self.tenant)
 
+    def create_all_tenants_filter_file(self):
```

Comment:
Too much duplicate code with 'update_filtering_file'-method here, please rewrite it, move duplicated parts to separate method, or just make one method with parameters for different tenants or all tenants in one.